README.md

```diff
@@ -18,8 +18,21 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+By default spree ember will use relative urls for the spree product images. 
+
+If server serving the assets is somewhere other than the same place as your ember app
+
+make sure `config.action_controller.asset_host` is set in your environment config
```

Comment:
Technically - this gem is designed for other use cases other than Spree Ember, so can we change this paragraph to:

Spree uses Paperclip, which works in the model scope, and therefore knows nothing about your hostname.  If you'd like the Image Serializer to output absolute URLs (requisite for Spree Ember), you'll need to set a `config.action_controller.asset_host` in your Rails environment configuration.